schedule/yast/textmode/textmode_default.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
let's provide postfix name `_svirt_xen.yaml` (or the variation you prefer for now :) ) instead of default.

Reply:
why there are two files for textmode?

Reply:
Renamed my file to `textmode_svirt_xen_pv.yaml`, the `textmode_svirt_xen` file is the old scheduler and maybe still needed for svirt-xen-hvm backend. I will see when I work on svirt-xen-hvm.